simapp/.goreleaser.yml

```diff
@@ -0,0 +1,34 @@
+project_name: simd
+
+release:
+  disable: false
+  name_template: "{{.Tag}}"
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - main: ./simd
+    goos:
+      - linux
+      - windows
```

Comment:
will windows builds work?  this might be an issue - not sure

Reply:
not sure actually. ill just remove it for now. i wont need it [remove windows](https://github.com/cosmos/cosmos-sdk/pull/23922/commits/3451c54f5f5b30cfcd8243a1cf33edfb31bea114)